Documentation/ceph-monitoring.md

```diff
@@ -81,7 +81,11 @@ You can find Prometheus Consoles here: https://github.com/ceph/cephmetrics/tree/
 A guide to how you can write your own Prometheus consoles can be found on the official Prometheus site here: https://prometheus.io/docs/visualization/consoles/.
 
 ## Prometheus Alerts
-To enable prometheus alerts during cluster deployment, make following changes to `cluster.yaml`
+To enable prometheus alerts during cluster deployment,
+```BASH
+kubectl create -f monitoring-roles.yaml
```

Comment:
Following the trend set by CSI, this should probably be renamed `monitoring-rbac.yaml`. I think that is a little more straightforward for people inspecting the directory as well.

Reply:
ack.